ui/receive_page.go

```diff
@@ -0,0 +1,594 @@
+
+	return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+		layout.Flexed(1, func(gtx C) D {
+			pg.receiveAddress.Text = pg.addrs
+			pg.receiveAddress.Alignment = text.Middle
+			card.Radius.NE = 8
+			card.Radius.SW = 8
+			card.Radius.NW = 0
+			card.Radius.SE = 0
+			return card.Layout(gtx, func(gtx C) D {
+				gtx.Constraints.Min.X = gtx.Constraints.Max.X
+				return layout.UniformInset(values.SMarginPadding16).Layout(gtx, func(gtx C) D {
+					return pg.receiveAddress.Layout(gtx)
+				})
+			})
+		}),
+		layout.Rigid(func(gtx C) D {
+			return layout.Inset{Left: values.MarginPadding1}.Layout(gtx, func(gtx C) D { return layout.Dimensions{} })
+		}),
+		layout.Rigid(func(gtx C) D {
+			card.Radius.NE = 0
+			card.Radius.SW = 0
+			card.Radius.NW = 8
+			card.Radius.SE = 8
+			return card.Layout(gtx, func(gtx C) D {
+				return pg.copy.Layout(gtx)
+			})
+		}),
+	)
+}
+
+func (pg *receivePage) Handle(common pageCommon) {
+	if pg.backdrop.Clicked() {
+		pg.isNewAddr = false
+	}
+
+	if pg.more.Button.Clicked() {
+		pg.isNewAddr = !pg.isNewAddr
+		if pg.isInfo {
+			pg.isInfo = false
+		}
+	}
+
+	if pg.newAddr.Button.Clicked() {
+		wallet := common.info.Wallets[*common.selectedWallet]
+		account := common.info.Wallets[*common.selectedWallet].Accounts[*common.selectedAccount]
+
+		addr, err := common.wallet.NextAddress(wallet.ID, account.Number)
+		if err != nil {
+			log.Debug("Error generating new address" + err.Error())
+		} else {
+			common.info.Wallets[*common.selectedWallet].Accounts[*common.selectedAccount].CurrentAddress = addr
+			pg.isNewAddr = false
+		}
+	}
+
+	if common.subPageInfoButton.Button.Clicked() {
+		go func() {
+			common.modalReceiver <- &modalLoad{
+				template:   ReceiveInfoTemplate,
+				title:      "Receive DCR",
+				cancel:     common.closeModal,
+				cancelText: "Got it",
+			}
+		}()
+	}
+
+	if common.subPageBackButton.Button.Clicked() {
+		*common.page = PageOverview
+	}
+
+	if pg.copy.Button.Clicked() {
+		go func() {
+			common.clipboard <- WriteClipboard{Text: common.info.Wallets[*common.selectedWallet].Accounts[*common.selectedAccount].CurrentAddress}
+		}()
+		pg.copy.Text = "Copied!"
+		pg.copy.Color = common.theme.Color.Success
+		time.AfterFunc(time.Second*3, func() {
+			pg.copy.Text = "Copy"
+			pg.copy.Color = common.theme.Color.Primary
+		})
+		return
+	}
+}
+
+func (wg *walletAccountWidget) accountSelectSection(gtx layout.Context, common pageCommon) layout.Dimensions {
+	border := widget.Border{
+		Color:        common.theme.Color.BorderColor,
+		CornerRadius: values.MarginPadding8,
+		Width:        values.MarginPadding2,
+	}
+	return border.Layout(gtx, func(gtx C) D {
+		return layout.UniformInset(values.MarginPadding12).Layout(gtx, func(gtx C) D {
+			return decredmaterial.Clickable(gtx, wg.fromAccountBtn, func(gtx C) D {
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						accountIcon := common.icons.accountIcon
+						accountIcon.Scale = 0.9
```

Comment:
The exact size of the icon should be used with a scale of 1.0. There seems to be an issue with gio when scaling below 1.0 which causes the image to get clipped on displays with low screen density.